src/openforms/js/formio_module.js

```diff
@@ -8,6 +8,7 @@ import PhoneNumberField from './components/form/phoneNumber';
 import BsnField from './components/form/bsn';
 import PostcodeField from "./components/form/postcode";
 import FileField from "./components/form/file";
+import SelectField from "./components/form/select";
```

Comment:
```suggestion
import SelectField from './components/form/select';
```

Reply:
Since we're using both single and double quotes in this file I wasn't sure which was the correct :p. I think it's worth changing all of them to single quotes in this file so they're all consistent.